### PR TITLE
Fix a build failure resulting from a race condition between two PRs.

### PR DESCRIPTION
**Pull request by abertelrud**
Each passed successfully but one unintentionally restored an import that the other had removed.

These were https://github.com/apple/swift-package-manager/pull/3029 and https://github.com/apple/swift-package-manager/pull/3019

### CODE REVIEW

**Comment by abertelrud** [comment]
@swift-ci please smoke test

**Review by neonichu — APPROVED** [review]

**Review by mattt — APPROVED** [review]

**Comment by abertelrud** [comment]
While we're waiting for builds to finish so I can merge this:  what's the best remedy for this kind of "PRs passing in the night"?  In this case both PRs' builds passed, but there was a conflict when landing.  I suppose I could have kicked off one more build right before merging, the problem is that it takes so long and there's very rarely a problem, so it's a low return on investment.

**Comment by DougGregor** [comment]
I'm going to go ahead and merge to unblock #3037

**Pull request merged**
